DQM/DTMonitorModule/interface/DTCalibValidationFromMuons.h

```diff
@@ -0,0 +1,117 @@
+
+#ifndef DTCalibValidationFromMuons_H
+#define DTCalibValidationFromMuons_H
+
+/** \class DTCalibValidationFromMuons
+ *  Analysis on DT residuals to validate the kFactor
+ *
+ *
+ *  \author G. Mila - INFN Torino
+ */
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "DataFormats/MuonDetId/interface/DTSuperLayerId.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include <FWCore/Framework/interface/EDAnalyzer.h>
+#include <FWCore/Framework/interface/LuminosityBlock.h>
+
+#include <DQMServices/Core/interface/DQMStore.h>
+#include <DQMServices/Core/interface/MonitorElement.h>
+#include <DQMServices/Core/interface/DQMEDAnalyzer.h>
+
+#include "DataFormats/MuonDetId/interface/DTWireId.h"
+#include "DataFormats/DTRecHit/interface/DTRecHitCollection.h"
+#include "DataFormats/DTRecHit/interface/DTRecSegment2DCollection.h"
+#include "DataFormats/DTRecHit/interface/DTRecSegment4DCollection.h"
+
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "RecoMuon/TrackingTools/interface/MuonSegmentMatcher.h"
+
+
+#include <FWCore/Framework/interface/ESHandle.h>
+
+#include <string>
+#include <map>
+#include <vector>
+
+
+// To remove into CMSSW versions before 20X
+class DQMStore;
```

Comment:
Is this forward declaration required? Using `DQMStore` instance directly is forbidden. I don't see it being used directly in this PR, so perhaps the declaration could be removed?

Reply:
Hello, sorry for the delay in the response, after consulting with colleagues, we see no reason to have that line. We accept your suggestion to remove this declaration.